### DISCUSSION TITLE
Containerized 7/2 Split

### DISCUSSION THREAD

**Discussion by joshuamoorexyz** [srsran/srsRAN_Project#526] [General Help] [answered]
I am writing to understand if the containerization is complete and if so how one would get the Dockerfiles associated with each? Is there any guide to configuring this?

**Comment by P4sca1** [comment, marked as answer]
It is not supported yet but on the roadmap.
https://docs.srsran.com/projects/project/en/latest/general/source/2_features_and_roadmap.html
